openwisp_controller/config/admin.py

```diff
@@ -1026,11 +1017,12 @@ class DeviceGroupAdmin(MultitenantAdminMixin, BaseAdmin):
         'organization',
         'description',
         'templates',
+        'context',
         'meta_data',
         'created',
         'modified',
     ]
-    search_fields = ['name', 'description', 'meta_data']
+    search_fields = ['name', 'description', 'meta_data', 'context']
```

Comment:
Are you sure of this? We don't have `context` as search fields in the other models.
Looking through too many textfields could make the search slow if there's many rows in the DB.
For consistency I would remove it.